packages/SwingSet/src/controller/initializeSwingset.js

```diff
@@ -392,7 +391,12 @@ export async function initializeSwingset(
       // during config, we believe bundle.id, but not at runtime!
       return bundle;
     }
-    return computeBundleID(bundle).then(id => ({ ...bundle, id }));
+    const { endoZipBase64Sha512 } = bundle;
+    assert.typeof(endoZipBase64Sha512, 'string');
+    return {
+      ...bundle,
+      id: `b1-${endoZipBase64Sha512}`,
+    };
```

Comment:
I would like to eliminate this since the bundle ID is already implied in the bundle and I don’t think we should be getting here unless the bundle was vetted, so I’m looking into how to not need `addBundleID`.

Reply:
But `bundle.id` annotation turns out to be pervasive, so leaving this where I found it.

Reply:
Adding `.id` was quick and convenient, feel free to change the pattern. In general, I've waffled between passing around `{ id, bundle }` vs passing around `{ id, ...bundle }`. The latter allowed me to change less code, but that doesn't mean it's the cleanest way to do it.